styleguide/components/components-list/components-list.css

```diff
@@ -28,17 +28,17 @@
 
 .link:link,
 .link:visited {
-  color: var(--palette-neutral-mid-dark);
+  color: var(--mds-grey-54);
 }
 
 .current {
   composes: link;
-  border-left: var(--selectedBorderWidth) solid var(--palette-primary-base);
+  border-left: var(--selectedBorderWidth) solid var(--mds-brand-100);
   font-weight: bold;
   padding-left: 3px;
 }
 
 .current:link,
 .current:visited {
-  color: black;
+  color: var(--black);
```

Comment:
I still think it's kind of weird to define our own primitives variables over the exact same primitive values that are available as CSS named colors.

Reply:
I think it's a design choice: if we want to make black slight less black then we can do it through variables. I'm not sure how much value that actually gives in modern times but it's been a thing. Maybe that's feedback for the design team.